duration duration
---
bool success
string message
